plugins/ltac2_ltac1_common/g_ltac2_ltac1_common.mlg

```diff
@@ -0,0 +1,71 @@
+(************************************************************************)
+(*         *   The Coq Proof Assistant / The Coq Development Team       *)
+(*  v      *         Copyright INRIA, CNRS and contributors             *)
+(* <O___,, * (see version control and CREDITS file for authors & dates) *)
+(*   \VV/  **************************************************************)
+(*    //   *    This file is distributed under the terms of the         *)
+(*         *     GNU Lesser General Public License Version 2.1          *)
+(*         *     (see LICENSE file for the text of the license)         *)
+(************************************************************************)
+
+DECLARE PLUGIN "coq-core.plugins.ltac2_ltac1_common"
```

Comment:
I find this name to be quite a mouthful. Couldn't the plugin and all related modules called `ltac_common` instead of `ltac2_ltac1_common`?